### PR TITLE
style sidebar logos

### LINKED ISSUE #177: we need Logos and text on the bottom of the side bar
We need to include a text (“Provided with the financial support of” Font: Arial or Helvetica) above the EU and German Cooperation logo and we need to adjust the ratio between the GIZ logo and the other two logos. 1:0.8 as this: 
grafik
The result should look similar to this: 
grafik

### PR DESCRIPTION
**Pull request by bmlancien**
Fix #177 
@Bachibouzouk Basically the logos are between 8 and 10 rem wide depending on the width of the sidebar (I set 45% width for each logo). Knowing that, can you use the function you built to set the size of the GIZ logo?

### CODE REVIEW

File: app/static/css/maps.css [hunk 1]
```diff
@@ -446,12 +446,23 @@ button {
...
+#footer-maps .footer-maps__text p {
+  font-size: 1rem;
+  padding-left: 0;
 }
 
-#footer-maps-logos-giz img{
-    text-align: right;
-    height: 4.09rem;
+#footer-maps-logos-eu-ger img,
+#footer-maps-logos-giz img {
```
Comment (Bachibouzouk) [inline app/static/css/maps.css]: The height ratio between the german cooperation and the giz logo has to be very specific. I can get what is the constraint for a given width if you need. Right now I have the ratio for a given height

File: app/templates/index.html [hunk 2]
```diff
@@ -240,15 +240,15 @@ <h2>Nigeria Se4All</h2>
         </div>
```
Comment (Bachibouzouk) [inline app/templates/index.html:239]: The header is missing or do you make in in another PR?

**Comment by Bachibouzouk** [comment]
> Knowing that, can you use the function you built to set the size of the GIZ logo?

Yes if you can guarantee that if the width of the giz-coop logo is set to say w, the width of the the giz logo will also be w (and not another number between 8 and 10 rem). We need to be able to fix the ratio of the width if you wish.

**Comment by Bachibouzouk** [comment]
> Knowing that, can you use the function you built to set the size of the GIZ logo?

The width of the giz logo should be 0.687 of the width of the giz cooperation logo
https://github.com/rl-institut/NESP2/issues/177#issuecomment-594188755

File: app/templates/index.html (hunk 2 above)
Comment (bmlancien) [reply app/templates/index.html:239]: It is missing, I didn't understant that it should be part of the footer too.

Comment (Bachibouzouk) [reply app/templates/index.html:239]: Not part of the footer, but part of a header

Comment (Bachibouzouk) [reply app/templates/index.html:239]: It is defined already but hidden

Comment (bmlancien) [reply app/templates/index.html:239]: Alright

Comment (bmlancien) [reply app/templates/index.html:239]: I added the logos to the header

File: app/static/css/maps.css (hunk 1 above)
Comment (bmlancien) [reply app/static/css/maps.css]: I used the ratio you mentioned in #177

Comment (Bachibouzouk) [reply app/static/css/maps.css]: I took a printscreen and checked: it respected the ratio :)

File: app/static/js/sidebar.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ var options = {
   ]
 };
 var map = L.map("map", options);
+map.fitBounds(L.latLngBounds(L.latLng(15, 17), L.latLng(2.4, 0.85)))
```
Comment (Bachibouzouk) [inline app/static/js/sidebar.js:14]: @christian-rli - I did this because the initial view is too much zoomed in for small screens. However if I zoom in, I can't zoom back out due to zoom restrictions. Do you have an idea on how to set this ?

**Review by Bachibouzouk — APPROVED** [review]
Thanks

File: app/static/js/sidebar.js (hunk 3 above)
Comment (christian-rli) [reply app/static/js/sidebar.js:14]: options.minZoom and options.maxZoom are set when entering national level and state level. So if you'd like to change those for the national level, change those values in 
`function adapt_view_to_national_level()`
or initially.
